test/tests/cli.spec.js

```diff
@@ -11,9 +10,6 @@ const pWriteFile = pify(fs.writeFile);
 const pReadFile = pify(fs.readFile);
 const pUnlink = pify(fs.unlink);
 
-// this is a bit of a long running test...
-jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
-
```

Comment:
I'm thinking we should keep the timeout. It may have put there because it was failing CI. Also, not every developer has a really fast computer. It should be replaced with

```js
jest.setTimeOut(20000);
```

Reply:
Sure.